swagger/v1/openapi.json

```diff
@@ -415,7 +415,7 @@
                   "relationships": {
                     "account": {
                       "data": {
-                        "id": "cf8ae1be-2626-4c7b-8134-bda1c78946c6",
+                        "id": "6a888174-88fd-4e80-a100-25550e5f77e6",
```

Comment:
Due to the nature of generated uuids by postgres, autogenerated examples from our fixtures will update uuids like this. I am not sure if we can work around this somehow, perhaps by setting the uuid manually for fixtures. It's a problem to solve for another day :)